pkg/scheduler/framework/session_plugins.go

```diff
@@ -209,8 +214,8 @@ func (ssn *Session) Preemptable(preemptor *api.TaskInfo, preemptees []*api.TaskI
 				victims = intersection
 			}
 		}
-		// Plugins in this tier made decision if victims is not nil
-		if victims != nil {
+		// Plugins in this tier made decision if victims is nil because intersection will be nil
+		if victims == nil {
```

Comment:
if victims is nil in first tier, the decision should be made by next tier, and so on... So l don't think you should change this.

Reply:
yeal, I have revet this one.